namespaces/verify-proxy-node-staging-test/deploy-pipeline.yaml

```diff
@@ -205,7 +205,7 @@ spec:
         - |
           set -euo pipefail
           cd /
-          bundle exec cucumber --strict --tags "not @ignore"
+          bundle exec cucumber --strict --tags "not @ignore" -f rerun --out rerun.txt || bundle exec cucumber --strict @rerun.txt
```

Comment:
concourse can retry failed tasks using the `attempts: 2` modifier to the task definition:

https://concourse-ci.org/attempts-step-modifier.html

I think this would be preferable over scripting it using `||` which may be harder to reason about

Reply:
I guess the argument was not to re-run all the tests but I don't have a strong preference either way